backend/src/main/java/gov/cdc/usds/simplereport/service/TestResultsDeliveryService.java

```diff
@@ -45,6 +45,7 @@ public boolean emailTestResults(PatientLink patientLink) {
     Map<String, Object> templateVariables =
         Map.of(
             "facility_name", patientLink.getTestOrder().getFacility().getFacilityName(),
+            "organization_name", patientLink.getTestOrder().getOrganization().getOrganizationName(),
```

Comment:
Does this require any update to our SendGrid template?

Reply:
we created a v2, will enable the new version after this goes out to prod